finetune/mtl.py

```diff
@@ -0,0 +1,152 @@
+import tensorflow as tf
+from finetune.base import BaseModel
+from finetune.input_pipeline import BasePipeline
+from finetune.errors import FinetuneError
+
+from finetune.sequence_labeling import SequenceLabeler
+from finetune.utils import indico_to_finetune_sequence
+
+
+def get_input_fns(tid, i_fn, v_fn):
```

Comment:
Would appreciate more descriptive variable names.  I get that these are meant to be input_fn and validation_fn but still think it's worth making things more explicit.